lib/src/interface.dart

```diff
@@ -242,3 +251,14 @@ abstract class SeltzerHttpResponse {
   /// Response payload.
   String get payload;
 }
+
+/// A message received by a [SeltzerWebSocket].
+abstract class SeltzerMessage {
```

Comment:
Done.  Add asynchronous callbacks because I can't find a way to parse a dart:html Blob synchronously.